src/main/java/ast/interfaces/ArrayDeclarationStatementDelegate.java

```diff
@@ -1,4 +0,0 @@
-package ast.interfaces;
```

Comment:
I've changed all files ending in "Delegate" to remove the word "Delegate" as I felt like this was adding clutter to the names.